keras_cv/layers/preprocessing/random_translation.py

```diff
@@ -193,17 +299,24 @@ def augment_bounding_boxes(
             target="rel_xyxy",
             images=image,
         )
-        bounding_boxes = RandomTranslation._translate_bounding_boxes(
+        bounding_boxes = RandomTranslation._transform_bounding_boxes(
             bounding_boxes,
             transformation["translate_horizontal"],
             transformation["translate_vertical"],
+            self.fill_mode,
         )
 
         bounding_boxes = bounding_box.clip_to_image(
             bounding_boxes,
             bounding_box_format="rel_xyxy",
             images=image,
         )
+        bounding_boxes = bounding_box.reject_empty(
```

Comment:
Thank you for this API; I actually REALLY like this API.

Reply:
@LukeWood I implemented this in 2 ways: 1st with ragged tensor outputs and 2nd with masking with sentinel values. I don't know which one would be preferred, but the ragged tensors didn't work with other elements of the data pipeline.